### PR TITLE
Vocabulary styles being overriden by custom styling 

### LINKED ISSUE #31: Vocabulary styles being overriden by custom styling
Due to issues with specificity, styling provided by CC Vocabulary is being overriden by styling from other third-party libraries and custom written CSS, leading to a different UI than what is expected. This is causing components to look broken or erratically styled.

**To Reproduce**
1. Check out the branch from PR #30
1. Run the development server

**Expected behavior**
The components should render like the examples in the Vocabulary [styleguide](https://creativecommons.github.io/cc-vocabulary/styleguide).

**Screenshots**
image
^ Broken `Footer` and `Locale`
image
^ Expected `Footer` and `Locale`

**Desktop (please complete the following information):**
 - OS: Linux (Fedora 30)
 - Browser: Firefox
 - Version: 68

### PR DESCRIPTION
**Pull request by onyijne**
Fixes #31 

**Description**

Fixed footer credits style

**Checklist:**
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -9,13 +9,17 @@
             <HTMLGenerator :shortLicenseName="selected.shortName"/>
         </div>
         <Footer>
-            <p>
-                <a href="https://github.com/creativecommons/cc-chooser">Icons</a> by The Noun Project and FontAwesome.
-            </p>
-            <p>
-                View this page's source code <a href="https://github.com/creativecommons/cc-chooser">on Github</a>.
```
Comment (akmadian) [inline src/App.vue]: Please add this link back into the footer, that’s the only thing I can see that I would like modified! I’ll take a look on my computer when I get home but this should be the last change before merge :)

Comment (akmadian) [reply src/App.vue]: To be clear, I mean the “View this page’s source code” link

Comment (onyijne) [reply src/App.vue]: Is the latest commit okay? Can I remove WIP?

Comment (akmadian) [reply src/App.vue]: Just realized, please remove the wikimedia commons link and I'll get this merged! Thanks :)

Comment (onyijne) [reply src/App.vue]: Done, thanks.

**Comment by akmadian** [comment]
Thanks, Onyijne!

**Pull request merged**